### PR TITLE
Updated Context API documentation

### PR DESCRIPTION
**Pull request by silvershadow82**
- Added `overrideContextOlderThanDays` parameter
- Updated language for easier understanding of the functionality.

### CODE REVIEW

File: spec/swagger.yaml [hunk 1]
```diff
@@ -96,6 +96,16 @@ tags:
...
     description: |-
       Visual context helps Translators make linguistic, layout, and spacing decisions based on where strings appear in your mobile or desktop application. The Context API supports uploading various forms of context (images and HTML), and associate the context with content in Smartling.
+      
+      **Context API Usage Limitations**:
+      * Maximum request size is 25MB.
+      * Maximum file upload size is 20MB.
+      * Maximum HTTP header size is 32KB.
+      * For bindings requests that use pagination, maximum page size is 5000 elements.
+      * For contexts requests that use pagination, maximum page size is 100 elements.
+      * Rate limits:
```
Comment (jones-smartling) [inline spec/swagger.yaml]: I would not specify API limits here like concurrency and max rate per sec (i think it should be removed from the others we have documented). The reason is we have a page that talks about receiving back a 429 and that they should wait and try again w/ an exponential retry. And I think it's better if clients use that approach rather then them trying to code under an API limit we might want to lower. That's my 2 cents.

Comment (silvershadow82) [reply spec/swagger.yaml]: UPDATED

File: spec/swagger.yaml [hunk 2]
```diff
@@ -1717,30 +1727,35 @@ paths:
...
           type: string
-          description: A part containing JSON object with an array of string hashcodes to match against.
-                       If the part content is missing, or empty array is passed, *all strings in the project* will be included in the match.
-                       This part contains the same parameters as a body of `/projects/{projectId}/contexts/{contextUid}/match/async` request.
       responses:
-        200:
-          description: OK
         202:
-          description: ACCEPTED
+          description: Request Accepted and processing started
```
Comment (enisher) [inline spec/swagger.yaml]: The description for 202 should be 'ACCEPTED' so it is consistent with other endpoints.

The asynchronous manner of the endpoint should be described in the endpoint description. See Jobs API for examples

File: spec/swagger.yaml [hunk 3]
```diff
@@ -1851,7 +1873,7 @@ paths:
           format: uuid
       responses:
         200:
-          description: OK
+          description: Successful response
```
Comment (enisher) [inline spec/swagger.yaml]: Response description for 200 should be `OK` to be consistent with other responses.

File: spec/swagger.yaml [hunk 4]
```diff
@@ -1894,25 +1916,28 @@ paths:
...
-          description: Unique identifier for the binding you want to delete
+          description: Unique identifier for the binding to be deleted
           in: path
           required: true
           type: string
           format: uuid
       responses:
         200:
-          description: OK
+          description: Successful response
```
Comment (enisher) [inline spec/swagger.yaml]: same here.

File: spec/swagger.yaml [hunk 5]
```diff
@@ -1939,32 +1964,35 @@ paths:
 
   /context-api/v2/projects/{projectId}/contexts/{contextUid}/bindings:
     get:
-      summary: Return all bindings for context
+      summary: Return all strings bindings for the context
```
Comment (enisher) [inline spec/swagger.yaml]: At some point @hchaps specifically removed articles from the summaries.

File: spec/swagger.yaml [hunk 6]
```diff
@@ -2041,11 +2070,14 @@ paths:
...
     post:
-      summary: Create bindings
+      summary: Create string to context bindings
       description: |-
-        Link context images with the strings they contain. The binding tells Smartling which context images to show translators for which strings.
-        A maximum of **150** bindings can be created per request. A request with more than **150** items will return a **400** response indicating validation error.
-        Basic bindings request should contain an object with "bindings" JSON array with at least one element (binding), featuring *contextUid* and *stringHashcode* properties:
+        Link context files with the strings they contain. 
+        The binding tells Smartling which context images to show translators for which strings.
+        A maximum of `150` bindings can be created per request.
```
Comment (enisher) [inline spec/swagger.yaml]: I think this is kind of a limit too. Do we want to avoid declaring it here too?

File: spec/swagger.yaml [hunk 7]
```diff
@@ -2105,27 +2141,27 @@ paths:
...
-          description: String-to-context JSON bindings
+          description: String-to-context bindings
           in: body
           required: true
           schema:
             $ref: '#/definitions/BindingsRequest'
       responses:
         200:
-          description: OK
+          description: Successful response
```
Comment (enisher) [inline spec/swagger.yaml]: Should be `OK` for consistency

File: spec/swagger.yaml (hunk 6 above)
Comment (silvershadow82) [reply spec/swagger.yaml]: Where else would we describe this limit?

File: spec/swagger.yaml (hunk 2 above)
Comment (silvershadow82) [reply spec/swagger.yaml]: UPDATED

File: spec/swagger.yaml (hunk 3 above)
Comment (silvershadow82) [reply spec/swagger.yaml]: UPDATED

File: spec/swagger.yaml (hunk 4 above)
Comment (silvershadow82) [reply spec/swagger.yaml]: UPDATED

File: spec/swagger.yaml (hunk 7 above)
Comment (silvershadow82) [reply spec/swagger.yaml]: UPDATED

**Pull request merged**